engines/gob/videoplayer.cpp

```diff
@@ -149,7 +149,7 @@ int VideoPlayer::openVideo(bool primary, const Common::String &file, Properties
 		}
 
 		// WORKAROUND: In Woodruff, Coh Cott vanished in one video on her party.
-		// This is a bug in video, so we work around it.
```

Comment:
"work around" is valid, and is the origin of the noun "workaround".
For all instances, "workaround" is not capitalized within an English sentence.